templates/docker-compose.yml

```diff
@@ -31,11 +32,14 @@ kubelet:
     - /var/lib/kubelet/:/var/lib/kubelet:rw
     - /var/run:/var/run:rw
     - {{manifest_directory}}:/etc/kubernetes/manifests:rw
+    - /srv/kubernetes:/srv/kubernetes
   command: |
     /hyperkube kubelet --containerized --hostname-override="{{private_address}}"
     --address="0.0.0.0" --api-servers=http://localhost:8080
-    --config=/etc/kubernetes/manifests {% if dns_server %} --cluster-dns={{dns_server}}
-    --cluster-domain=cluster.local {% endif %}
+    --config=/etc/kubernetes/manifests {% if dns_server %}
```

Comment:
if it's not causing a problem, leave this as is - but we may want to leverage the helpers in jinja to keep this line tidy

using a - inline in the if statement can control how spaces/newlines are rendered in the template

eg:

```
{%- if dns_server %}
...
{% endif -%}
```
that will keep everything bunched in place, and strip any emptylines that get rendered.

But if its not causing a problem, i wouldn't fiddle with it